knplabs/dictionary-bundle/2.0/config/packages/knp_dictionary.yaml

```diff
@@ -0,0 +1,17 @@
+knp_dictionary:
+    dictionaries: []
+
+# Examples
```

Comment:
Do we need this example docs? Could we do a link the documentation instead?

Reply:
Already removed :wink: